### PR TITLE
Update docs for the simulator

### PR DESCRIPTION
**Pull request by Adimote**
Update the docs for the simulator to match the new API and new robot layout.

(This gives hints about the new robot configuration, but we decided it's vague enough to keep public).

### CODE REVIEW

File: simulator/simulator_programming.md [hunk 1]
```diff
@@ -73,51 +73,56 @@ Your robot has one servo board attached, the jaws of the robot are controlled by
...
-| Pin | Location |
-|-----|----------|
-| 3   | Red (Left) |
-| 4   | Green (right) |
+| Pin | Location      | Required Mode |
+|-----|---------------|---------------|
+| 3   | Red (lower)   | `OUTPUT`      |
+| 4   | Green (upper) | `OUTPUT`      |
 
 Using the `digital_write` method, you can set these to True (On) or False (Off).
```
Comment (PeterJCLaw) [inline simulator/simulator_programming.md:127]: maybe?
```suggestion
Using the `digital_write` method, you can set these to `True` (On) or `False` (Off).
```

File: simulator/simulator_programming.md [hunk 2]
```diff
@@ -73,53 +73,58 @@ Your robot has one servo board attached, the jaws of the robot are controlled by
...
 
 #### Distance Sensors
 
-Analogous to ultrasound sensors, distance sensors allow you to retrieve the distance between your robot and an object. These are attached to analogue pins A0-A5:
+Analogous to ultrasound sensors, distance sensors allow you to retrieve the distance between your robot and an object. These are attached to analog pins A0-A5:
 
-| Pin | Location |
-|-----|----------|
-| A0  | Front Left |
+| Pin | Location    |
```
Comment (PeterJCLaw) [inline simulator/simulator_programming.md:106]: Should we have the required pin modes here too? (Presumably also `INPUT`?)

File: simulator/simulator_programming.md [hunk 3]
```diff
@@ -73,53 +73,58 @@ Your robot has one servo board attached, the jaws of the robot are controlled by
...
+The rear of the robot has a wide microswitch.
+
 The microswitch is attached to digital pin 2:
 
-| Pin | Location |
-|-----|----------|
-| 2   | Back     |
+| Pin | Location | Required Mode |
+|-----|----------|---------------|
+| 2   | Back     | `INPUT`       |
```
Comment (PeterJCLaw) [inline simulator/simulator_programming.md:98]: For info: `INPUT` turns out to be the default mode. I think it's fine to have this suggestion here though.

**Review by PeterJCLaw — APPROVED** [review]
Thanks!
Should we also specify the pin modes for the analogue pins?

File: simulator/simulator_programming.md (hunk 2 above)
Comment (Adimote) [reply simulator/simulator_programming.md:106]: I thought they didn't care what mode they were in?

Comment (Adimote) [reply simulator/simulator_programming.md:106]: turns out they need to be in `INPUT` at least

**Pull request merged**